Branch-SDK/src/io/branch/referral/util/CommerceEvent.java

```diff
@@ -137,6 +131,7 @@ public JSONObject getCommerceJSONObject() {
     }
 
     public List<JSONObject> getProducts() {
+        if (this.products == null) return null;
```

Comment:
Can we make this line safe instead: https://github.com/BranchMetrics/android-branch-deep-linking/blob/21ede340d9e03d99f4df68a85609b6774c5730ac/Branch-SDK/src/io/branch/referral/util/CommerceEvent.java#L125